src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import core.basesyntax.service.FileWriterService;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import core.basesyntax.service.impl.FruitShopServiceImpl;
+
+public class Main {
+    private static final String PATH = "src/main/java/core/basesyntax/db/data.csv";
+    private static final String REPORT = "src/main/java/core/basesyntax/db/report.csv";
+
+    public static void main(String[] args) {
+        FileWriterService writer = new FileWriterServiceImpl();
+        FruitShopService transactions = new FruitShopServiceImpl();
+        writer.writeToFile(transactions.fruitTransaction(PATH), REPORT);
+    }
```

Comment:
Your classes performs many things instead of concentration on single responsibility. Much more operations
should be here
1. Read content of file
2. Parse raw content to Transactions
3. Process transactions to change Storage state
4. Form report using new Storage state
5. Write report to file

And each of this logical steps should be performed by separate service


Please recheck task description after ```Hint``` section, there are some nice advises